### PR TITLE
Update protocol update block number on various contents

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

Update Istanbul hardfork block number on Cypress network. 
I also make an issue requesting a new page describing hardfork history and blocknumbers in https://github.com/klaytn/klaytn-docs/issues/419

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by kjhman21 — APPROVED** [review]

**Comment by aidan-kwon** [comment]
@terrikwak PTAL

File: docs/klaytn/design/computation/computation-cost/computation-cost.md [hunk 1]
```diff
@@ -17,16 +17,12 @@ Therefore, we chose the third option for Klaytn. For now, the limit of the execu
...
 If you want the previous document, please refer to [previous document](computation-cost-previous.md).
 
-v1.7.0 Protocol Upgrade - incompatible changes including the **Istanbul** hard fork items and Klaytn's own items. 
-It has been enabled from block number `#75373312` in case of Baobab network. 
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
-
-v1.7.3 Protocol Upgrade - incompatible changes including Base Fee from the **London** hard fork. 
-It has been enabled from block number `#80295291` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
+`IstanbulEVM` protocol upgrade block number is like the followings.
```
Comment (terrikwak) [inline docs/klaytn/design/computation/computation-cost/computation-cost.md:23]:
```suggestion
`IstanbulEVM` protocol upgrade block number is as follows.
```

File: docs/klaytn/design/computation/computation-cost/computation-cost.md [hunk 2]
```diff
@@ -17,16 +17,12 @@ Therefore, we chose the third option for Klaytn. For now, the limit of the execu
 The below table shows the computation cost of EVM opcodes. The computation cost was determined based on experiments.
 
 {% hint style="success" %}
-NOTE: Computation costs have been changed with the protocol upgrade, or the hard fork.
+NOTE: Computation costs have been changed with `IstanbulEVM` protocol upgrade, or the "hard fork".
```
Comment (terrikwak) [inline docs/klaytn/design/computation/computation-cost/computation-cost.md:20]:
```suggestion
NOTE: Computation costs have been changed with the `IstanbulEVM` protocol upgrade, or the "hard fork".
```

File: docs/klaytn/design/computation/klaytn-virtual-machine/klaytn-virtual-machine.md [hunk 3]
```diff
@@ -93,16 +93,12 @@ Storage fees have a slightly nuanced behavior. To incentivize minimization of th
...
 If you want the previous document, please refer to [previous document](klaytn-virtual-machine-previous.md).
 
-v1.7.0 Protocol Upgrade - incompatible changes including **Istanbul** hard fork items and Klaytn's own items.
-It has been enabled from block number `#75373312` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
-
-v1.7.3 Protocol Upgrade - incompatible changes including BaseFee **London** Hard-Fork item.
-It has been enabled from block number `#80295291` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
+`IstanbulEVM` protocol upgrade block number is like the followings.
```
Comment (terrikwak) [inline docs/klaytn/design/computation/klaytn-virtual-machine/klaytn-virtual-machine.md:99]:
```suggestion
`IstanbulEVM` protocol upgrade block number is as follows.
```

File: docs/klaytn/design/transaction-fees/transaction-fees.md [hunk 4]
```diff
@@ -44,16 +44,12 @@ Klaytn currently does not provide a way to replace a transaction using the unit
 Basically, Klaytn is keeping compatibility with Ethereum. So Klaytn's gas table is pretty similar with that of Ethereum. But there are some features unique to Klaytn that require several new constants.
 
 {% hint style="success" %}
-NOTE: The gas table has changed with the protocol upgrade, or the hard fork.
+NOTE: The gas table has changed with `IstanbulEVM` protocol upgrade, or the "hard fork".
```
Comment (terrikwak) [inline docs/klaytn/design/transaction-fees/transaction-fees.md:47]:
```suggestion
NOTE: The gas table has changed with the `IstanbulEVM` protocol upgrade, or the "hard fork".
```

File: docs/klaytn/design/transaction-fees/transaction-fees.md [hunk 5]
```diff
@@ -44,16 +44,12 @@ Klaytn currently does not provide a way to replace a transaction using the unit
...
 If you want the previous document, please refer to [previous document](transaction-fees-previous.md).
 
-v1.7.0 Protocol Upgrade - incompatible changes including **Istanbul** hard fork items and Klaytn's own items.
-It has been enabled from block number `#75373312` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
-
-v1.7.3 Protocol Upgrade - incompatible changes including Base Fee from the **London** hard fork.
-It has been enabled from block number `#80295291` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
+`IstanbulEVM` protocol upgrade block number is like the followings.
```
Comment (terrikwak) [inline docs/klaytn/design/transaction-fees/transaction-fees.md:50]:
```suggestion
`IstanbulEVM` protocol upgrade block number is as follows.
```

File: docs/smart-contract/precompiled-contracts/precompiled-contracts.md [hunk 6]
```diff
@@ -6,15 +6,11 @@ The precompiled contracts from address 0x01 through 0x09 are the same as those i
 Klaytn additionally implements precompiled contracts from 0x3fd through 0x3ff to support new Klaytn features.
 
 {% hint style="success" %}
-NOTE: With the protocol upgrade, or the hard fork introduced in Klaytn v1.7.0, three precompiled contract addresses have changed, and **blake2F** was added.
+NOTE: Three precompiled contract addresses have changed, and **blake2F** was added after `IstanbulEVM` protocol upgrade, or the "hard fork".
```
Comment (terrikwak) [inline docs/smart-contract/precompiled-contracts/precompiled-contracts.md:9]:
```suggestion
NOTE: Three precompiled contract addresses have been changed, and **blake2F** was added after the `IstanbulEVM` protocol upgrade, or the "hard fork".
```

File: docs/smart-contract/precompiled-contracts/precompiled-contracts.md [hunk 7]
```diff
@@ -6,15 +6,11 @@ The precompiled contracts from address 0x01 through 0x09 are the same as those i
...
 
-v1.7.0 Protocol Upgrade - incompatible changes including **Istanbul** hark fork items and Klaytn's own items.
-It has been enabled from block number `#75373312` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
-
-v1.7.3 Protocol Upgrade - incompatible changes including Base Fee from the **London** hard fork.
-It has been enabled from block number `#80295291` in case of Baobab network.
-Cypress mainnet will be subject to the same protocol upgrade in the next version.
+`IstanbulEVM` protocol upgrade block number is like the followings.
```
Comment (terrikwak) [inline docs/smart-contract/precompiled-contracts/precompiled-contracts.md:11]:
```suggestion
`IstanbulEVM` protocol upgrade block number is as follows.
```

**Review by terrikwak — APPROVED** [review]

**Pull request merged**